docs/03-pages/02-api-reference/01-components/head.mdx

```diff
@@ -57,7 +59,15 @@ In this case only the second `<meta property="og:title" />` is rendered. `meta`
 
 > The contents of `head` get cleared upon unmounting the component, so make sure each page completely defines what it needs in `head`, without making assumptions about what other pages added.
 
+## Use minimal nesting
+
 `title`, `meta` or any other elements (e.g. `script`) need to be contained as **direct** children of the `Head` element,
 or wrapped into maximum one level of `<React.Fragment>` or arrays—otherwise the tags won't be correctly picked up on client-side navigations.
 
-> We recommend using [next/script](/docs/pages/building-your-application/optimizing/scripts) in your component instead of manually creating a `<script>` in `next/head`.
+## Use `next/script` for scripts
+
+We recommend using [next/script](/docs/pages/building-your-application/optimizing/scripts) in your component instead of manually creating a `<script>` in `next/head`.
```

Comment:
```suggestion
We recommend using [`next/script`](/docs/pages/building-your-application/optimizing/scripts) in your component instead of manually creating a `<script>` in `next/head`.
```